Application/registerationForm.js

```diff
@@ -0,0 +1,49 @@
+document.getElementById('registerationForm').addEventListener('submit', function (event) {
+  event.preventDefault();
+
+  if (emailCheck() && contactCheck()) {
+    alert("Registeration done sucessfull");
+    location.replace("home.html");
+  } else {
+    alert("Registeration not sucessfull");
+
+  }
+});
+
+function passwordShow() {
+  const password = document.getElementById("Password");
```

Comment:
The variable name should be `passwordElement` instead of `password`. Using passwordElement makes it clear that the variable is holding a reference to a DOM element, specifically an element with the ID "Password". This helps in maintaining code readability and avoiding potential confusion.